app/Services/Notification/index.ts

```diff
@@ -0,0 +1,3 @@
+import Notification from './Notification'
```

Comment:
Why not putting the `NotificationService` class directly in this file?

Reply:
I prefer this organization to export anything needed, such as types. Indeed, I need to export types from there, do not resolve the issue until I've done it